tasks/51_abbreviated-name/abbreviated-name.js

```diff
@@ -0,0 +1,14 @@
+export function abbreviated(fullName) {
+  return fullName.split(' ')
+      .map((name, i) => {
+        if (i !== 0) {
+          return name[0] + '.';
+        }
+        return name;
+      })
+      .join(' ');
+}
+// Answer using regEx, but not allowed by linter
+// export function abbreviated(fullName) {
+//   return fullName.replace(/(?<=\s[A-Z])[a-z]*/g, '.');
+// }
```

Comment:
I think that changing `"ecmaVersion"` in .eslintrc.json to 2018 will do the trick. Don't have to change it, this is just a hint.